09-student_guide.Rmd

```diff
@@ -0,0 +1,277 @@
+
+The fasta file we're loading first contains the aligned sequences for the spike protein of 5 SARS-CoV-2 samples. This is what the top of the file looks like in a text editor:
+
+<img src="resources/images/09-fasta.png" title="Examples of a fasta file format" alt="An example of the first few lines of a fasta file" style="display: block; margin: auto;" />
+
+We can load this file into RStudio using the `read.FASTA` command and save it as the object "spike.small".
+
+```{r, message = FALSE, warning = FALSE}
+spike.fasta <- read.FASTA("sars_spike_protein.fasta")
+```
+
+After we've created an object in RStudio, we can get information about the object by typing the object's name.
+
+```{r, message = FALSE, warning = FALSE}
+spike.fasta
+```
+
+Here we can see a summary of what this object contains, as well as how long the sequences are and the sequence names. 
+
+Notice that RStudio has saved the information in the fasta file as binary data. This means the sequence information has been converted from "ATCG" into something easier for RStudio to work with, but harder for humans to interpret.
+
+The `phangorn` package uses a special data format called `phyDat`, which is derived from the fasta format. A `phyDat` object provides some additional information about the samples we upload. 
+
+```{r, message = FALSE, warning = FALSE}
+spike.phydat <- read.phyDat("sars_spike_protein.fasta", format = "fasta")
+
+spike.phydat
+```
+
+Different site patterns refers to sites that differ between sequences. In this small dataset, 46 of the 3827 possible bases (characters) show differences among these SARS-CoV-2 samples.
+
+We've been working with a dataset that contains the original SARS-CoV-2 sequence (the Wuhan reference sample), as well as samples of the alpha, beta, delta, and gamma variants. Another variant, the omicron variant, was first identified in late 2021 and quickly became a variant of concern. Let's look at a dataset that contains additional omicron samples.
+
+```{r, message = FALSE, warning = FALSE}
+spike.omicron <- read.phyDat("sars_spike_protein_omicron.fasta", format = "fasta")
+
+spike.omicron
+```
+
+::: {.reflection}
+QUESTIONS:
+
+1. What is some information saved in a .fasta object that RStudio tells us that we don't get from a .phyDat object?
+
+2. How many omicron sequences are there in the second file (loaded into the object `spike.omicron`)?
+
+3. Do you think there is more variability in the omicron sequences than in other variants (alpha, beta, delta, and gamma)? Why or why not?
+:::
+
+## Exercise Three: How are variants related to the original SARS-CoV-2 reference strain?
+
+We'll work with both the smaller spike protein dataset (saved as  `spike.small`) and the spike protein dataset with omicron samples (saved as `spike.omicron`) in this activity. 
```

Comment:
isn't it saved as `spike.phydat`? This could be a tad confusing

Reply:
Bah, thank you for catching this!